static/app/utils/useOwners.tsx

```diff
@@ -0,0 +1,72 @@
+import {useEffect, useMemo} from 'react';
+
+import {useMembers} from 'sentry/utils/useMembers';
+import {useTeams} from 'sentry/utils/useTeams';
+import {useTeamsById} from 'sentry/utils/useTeamsById';
+
+interface Options {
+  /**
+   * The current selected values that will be ensured loaded. These should be
+   * in the actor identifier format
+   */
+  currentValue?: string[];
+}
+
+/**
+ * Hook to fetch owner options
+ */
+export function useOwners({currentValue}: Options) {
+  // Ensure the current value of the fields members is loaded
+  const ensureUserIds = useMemo(
+    () =>
+      currentValue?.filter(item => item.startsWith('user:')).map(user => user.slice(7)),
```

Comment:
`slice(7)`? what are we trying to do, maybe a `replace()` call would be clearer

Reply:
we would probably want to use a regex replace to make sure it trims from the fromt, but I think this is clear enough IMO

Reply:
the mysterious part is that `user:` is only 5 chars, but we're stripping off 7.

Reply:
👀 that sounds like a ream bug lol